cluster-autoscaler/cloudprovider/packet/packet_manager_rest.go

```diff
@@ -332,8 +332,8 @@ func createPacketManagerRest(configReader io.Reader, discoverOpts cloudprovider.
 			cfg.Nodegroupdef[nodepool].ClusterName = opts.ClusterName
 		}
 
-		manager.packetManagerNodePools[nodepool] = &packetManagerNodePool{
-			baseURL:           "https://api.packet.net",
+		manager.equinixMetalManagerNodePools[nodepool] = &equinixMetalManagerNodePool{
+			baseURL:           "https://api.equinix.com",
```

Comment:
The base URL must include a `/metal/v1` prefix

Reply:
Done